### PR TITLE
Control PopoverContent props through propsConfigs in SingleDatepicker

### PR DESCRIPTION
**Pull request by stieludv**
My suggested fix.
First SingleDatepicker in example showcases this functionality.

### CODE REVIEW

File: example/index.tsx [hunk 1]
```diff
@@ -37,6 +37,12 @@ const App = () => {
           minDate={new Date('07/25/2022')}
           maxDate={new Date('08/05/2022')}
           onDateChange={setDate}
+          propsConfigs={{
+            popoverProps: {
+              bg: "red",
```
Comment (stieludv) [inline example/index.tsx:42]: bg: "black",
color: "white";

**Comment by aboveyunhai** [comment]
let me take a look at it tonight.

**Comment by aboveyunhai** [comment]
@stieludv I changed the `popoverProps` to `popoverComp(onent)Props` because `PopoverProps` has specific meaning for Chakra.
so the following should fit your need, it's verbose but unambiguous
```ts
  popoverCompProps: {
     popoverContentProps { 
      // your style here
     }
  }
```

Other than that, thanks for your work!

**Comment by aboveyunhai** [comment]
0.2.5 is out!

**Pull request merged**